financial-connections/src/main/java/com/stripe/android/financialconnections/features/accountpicker/AccountPickerViewModel.kt

```diff
@@ -180,7 +181,7 @@ internal class AccountPickerViewModel @Inject constructor(
         )
     }
 
-    fun onAccountClicked(account: PartnerAccount) = withState { state ->
```

Comment:
What do you think about adding a `withState` method to our base ViewModel?

Reply:
https://github.com/stripe/stripe-android/pull/8154/commits/38c29bc8dc5fb45b0fab966180605b70dc49de6f